infra/azure/terraform/README.md

```diff
@@ -0,0 +1,62 @@
+# infra/azure
+Terraform scripts and documentation for Cluster API for Azure ([CAPZ](https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/main/templates)) infrastructure that the Kubernetes community runs on Azure.
+The capz folder structure is formatted shown below:
+main.tf
+├── resource_group
+├── identities
+├── key_vault
+├── container_registry
+├── storage_account
+├── role_assignments
+├── variables
+
+# Prerequiste 
+The `az-cli-prow` service principal with federated credentials is required to be created by a tenant admin. The [iam-config\gce-param.json](./iam-config/gce-param.json) and [iam-config\eks-param.json](./iam-config/eks-param.json) can be used for creating the fededrated credentials for GCE and EKS. Below is an example of how to add a federated credential.
+
+    ```sh
+    az ad sp create-for-rbac --name az-cli-prow
+    appid=$(az ad sp list --filter "displayName eq 'az-cli-prow'" --query [0].appId --output tsv)
+    az ad app federated-credential create --id $appid --parameters param.json 
+    ```
+
+The service principal needs the below mentioned role assignements, which gets taken care of by [role-assignments\main.tf](.role-assignments/main.tf): 

```

Comment:
```suggestion
The service principal needs the below mentioned role assignements, which gets taken care of by [role-assignments/main.tf](.role-assignments/main.tf): 

```
I think it's confusing to use backslashes for paths here--this is documentation for Linux, right?